sdk/python/feast/entity.py

```diff
@@ -30,8 +30,25 @@
 class Entity:
     """
     Represents a collection of entities and associated metadata.
+
+    Args:
+        name: Name of the entity.
+        value_type (optional): The type of the entity, such as string or float.
+        description (optional): Additional information to describe the entity.
+        join_key (optional): A property that uniquely identifies different entities
+            within the collection. Used as a key for joining entities with their
+            associated features. If not specified, defaults to the name of the entity.
+        labels (optional): User-defined metadata in dictionary form.
     """
 
+    _name: str
+    _value_type: ValueType
+    _description: str
+    _join_key: str
+    _labels: MutableMapping[str, str]
+    _created_timestamp: Optional[Timestamp]
+    _last_updated_timestamp: Optional[Timestamp]
```

Comment:
Similar comment, we should be using `datetime` instead of `Timestamp`.

Reply:
Got it.